src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprints.java

```diff
@@ -23,6 +23,15 @@ private static String trim(String imageId) {
         }
         return imageId.substring(0, 32);
     }
+    
+    /**
+     * Gets a fingerprint hash for Docker image.
+     * @param imageId Docker image ID.
+     * @return 32-symbol fingerprint hash
+     */
+    /*package*/ static @Nonnull String getImageFingerprintHash(@Nonnull String imageId) {
+        return trim(imageId);
```

Comment:
I'll rework it in a separate pull request, because it needs an additional discussion regarding the public access modifier.